packages/tabs/README.md

```diff
@@ -103,7 +142,7 @@ import {
 </sp-tab-panel>
 </sp-tabs>
 
-## Horizontal Tabs
+#### Variants
 
 An `<sp-tabs>` element will display horizontally by default. It can be modified with states like `compact`, `disabled`, and `quiet`, or with content like icons, etc.
```

Comment:
```suggestion
An `<sp-tabs>` element will display horizontally by default. It can be modified with states like `compact`, and `quiet`, or with content like icons, etc.
```